amy/workshops/models.py

```diff
@@ -2876,6 +2876,32 @@ class CommonRequest(SecondaryEmailMixin, models.Model):
         verbose_name="Department/School/Library affiliation (if applicable)",
     )
 
+    MEMBER_AFFILIATION_CHOICES = (
+        ("yes", "Yes"),
+        ("no", "No"),
+        ("unsure", "I'm not sure"),
+    )
+    member_affiliation = models.CharField(
```

Comment:
Why is this field needed at all? Will it be used for something else than making `member_code` required or not?

Because if we want to have a soft validation, we don't really need this field.

Reply:
The intent is for the `member_code` to eventually be displayed dynamically if the user marks Yes to the affiliation question. But depending on how we decide to approach validation, it's true that we may not need this question in the end.